src/main/java/org/ideoholic/curium/model/feescategory/action/FeesAction.java

```diff
@@ -71,8 +73,7 @@ public String feesConcessionReport() {
 
 	@PostMapping("/waiveOffFees")
 	public String waiveOffFees() {
-		String studentId = feesActionAdapter.waiveOffFees();
```

Comment:
Why was this line removed
`String studentId = feesActionAdapter.waiveOffFees();`